app/models/users/user.rb

```diff
@@ -48,7 +48,8 @@ def self.authenticate(login, password)
   end
 
   # Creates a new user based on the auth data passed from OmniAuth.
-  def self.create_with_omniauth(auth)
+  def self.create_with_omniauth(auth, subdomain = nil)
+    return false if subdomain == 'admin' || subdomain.blank?
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.